autoload/nnn.vim

```diff
@@ -366,6 +366,7 @@ function! nnn#pick(...) abort
         let l:sess_cfg = ' '
     endif
 
+    let g:nnn#command = 'NNN_SEL='.shellescape(s:temp_file).' '.g:nnn#command
     let l:cmd = g:nnn#command.l:sess_cfg.' -p '.shellescape(s:temp_file).' '.(l:directory != '' ? shellescape(l:directory): '')
```

Comment:
I think `g:nnn#command` shouldn't be modified. Can it be appended to `l:cmd` instead?

Reply:
I think the cleanest solution would be to use the `env` options when creating a terminal buffer. Note that vim and neovim has different APIs but both accepts a map of name:value.
For vim: `:h job-options`. For neovim `:h jobstart()`.

Reply:
> I think `g:nnn#command` shouldn't be modified. Can it be added to `l:cmd` instead?

Just like `let l:cmd = 'NNN_SEL='.shellescape(s:temp_file).' '.g:nnn#command.l:sess_cfg.' -p '.shellescape(s:temp_file).' '.(l:directory != '' ? shellescape(l:directory): '')
`?
It also works too.

> use the env options when creating a terminal buffer.

Yes, I think so too. I need to figure out how to add it to `env` options:)

Reply:
Take a look at the `create_term_buf` where the terminal buffer is created. You should be able to pass the env from there into `termopen` and `term_start`.

Reply:
Sorry for my late reply. I moved it to `env` and tested. It works both in vim and nvim.